vmm/src/vm.rs

```diff
@@ -73,6 +74,11 @@ const KERNEL_64BIT_ENTRY_OFFSET: u64 = 0x200;
 const IOAPIC_RANGE_ADDR: u64 = 0xfec0_0000;
 const IOAPIC_RANGE_SIZE: u64 = 0x20;
 
+// Maximum number of signals
+const MAX_SIGNAL_VAL: usize = 64;
+// Identifier for terminal window size
+const TIOCGWINSZ: u64 = 21523;
```

Comment:
No need to create a constant for this, you should instead use the one from libc. `libc::TIOCGWINSZ`

Reply:
Agreed.